runtime/v2/app.go

```diff
@@ -65,6 +66,10 @@ type App[T transaction.Tx] struct {
 	interfaceRegistrar registry.InterfaceRegistrar
 	amino              legacy.Amino
 	moduleManager      *MM[T]
+
+	// GRPCQueryDecoders maps gRPC method name to a function that decodes the request
+	// bytes into a gogoproto.Message, which then can be passed to appmanager.
+	GRPCQueryDecoders map[string]func(requestBytes []byte) (gogoproto.Message, error)
```

Comment:
these are registereed from modules exposing gRPC queries and map gRPC method=>something that decodes the req.